### PR TITLE
Add cool demo for heading property

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**
screenshot

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@ebidel PTAL. :wink:

File: demo/navigation-arrow.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- Copyright Eric Bidelman <ebidel@gmail.com> -->
```
Comment (ebidel) [inline demo/navigation-arrow.html]: This is all you :)

**Comment by ebidel** [comment]
Nice. LGTM.

**Comment by FluorescentHallucinogen** [comment]
@ebidel Could you please release the new version `2.0.2` for these changes to take effect on https://webcomponents.org?

**Comment by ebidel** [comment]
Done

**Comment by FluorescentHallucinogen** [comment]
@ebidel Just noticed that when I open the demo, the following error occurs in the browser console:

`_stampTemplate @ template-stamp.html:425 Error: <polygon> attribute transform: Expected number, "rotate()".`

Seems this is because of updating the `heading` value in `transform$="rotate([[heading]])"` via data binding:

```svg
<svg viewBox="-4.5625 -4.5625 9.125 9.125" width="100" height="100">
  <polygon
      points="0,-4.5625 3,3.4375 0,1.4375 -3,3.4375"
      fill="#db4437"
      transform$="rotate([[heading]])">
  </polygon>
</svg>
```

The `heading` value can be `undefined`/`null` during the web component initialization. Moreover, according to the W3C Geolocation API spec, if the implementation cannot provide heading information, the value of this attribute must be `null`. If the hosting device is stationary (i.e. the value of the `speed` attribute is 0), then the value of the `heading` attribute must be `NaN`.

I believed that setting the default value of `heading` property to 0, should solve this problem.

This is not good. How to fix it?

**Comment by ebidel** [comment]
The compass should point North until there's a value.

**Comment by FluorescentHallucinogen** [comment]
@ebidel 
> The compass should point North until there's a value.

It do! :)

Seems, you didn't understand what I meant in my previous comment. Please open https://raw-dot-custom-elements.appspot.com/ebidel/geo-location/2.0.2/geo-location/demo/index.html page and check errors in the browser console.

**Comment by FluorescentHallucinogen** [comment]
@ebidel Ping.

**Pull request merged**